organic_scoring/synth_dataset/synth_dataset_base.py

```diff
@@ -0,0 +1,7 @@
+from abc import ABC
+from typing import Any
+
+
+class SynthDatasetBase(ABC):
+    def sample(self) -> Any:
```

Comment:
again, probably should be abstract

```suggestion
from abc import ABC, abstractmethod
...

    @abstractmethod
    def sample(self) -> Any:
```